modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -757,6 +757,64 @@ test.describe('Manage object fields through Model Builder', () => {
 });
 
 test.describe('Manage objectFields through Objects Admin UI', () => {
+	test("can check that it's possible to update created custom fields in a System Object", async ({
```

Comment:
This test is just creating a custom field, so we can use the term in the singular. Additionally, we can use the object prefix and use the term "System Object" in lowercase. Simplifying a bit we have:

`can update custom object field in a system object`